graylog2-web-interface/src/components/sidecars/configuration-forms/ConfigurationHelper.jsx

```diff
@@ -8,12 +8,12 @@ import DocsHelper from 'util/DocsHelper';
 
 import FilebeatHelper from './FilebeatHelper';
 import TemplatesHelper from './TemplatesHelper';
-import IncludesHelper from './IncludesHelper';
+import ConfigurationVariablesHelper from './ConfigurationVariablesHelper';
 import ConfigurationHelperStyle from './ConfigurationHelper.css';
 
 class ConfigurationHelper extends React.Component {
   static propTypes = {
-    type: PropTypes.string.isRequired,
+    variableRenameHandler: PropTypes.func.isRequired,
```

Comment:
I would rename this to better match our most usual conventions: `onVariableRename` would be the prop name, `variableRenameHandler` would be the function that handles the "event" in the end.